### ISSUE TITLE
Nautilus sidebar question

### ISSUE THREAD

**Issue by froggy-jpg** [vinceliuice/WhiteSur-gtk-theme#1182]
i have a question regarding nautilus, in screenshot of other people (on the left) you can see that the top of the sidebar is empty except for the window controls, however in my case (on the right) there is a really annoying menu to the right of window controls

could this be because of the gnome version im using? i am at gnome 47, if this is relevant

![image](https://github.com/user-attachments/assets/58d46a1b-7fca-4d2e-b5ee-6080dc817a75)

**Comment by vinceliuice** [comment]
Left one is old nautilus version

**Comment by froggy-jpg** [comment]
i see, could there possibly be a way around that? because of it the sidebar looks massive

**Comment by vinceliuice** [comment]
Nope, new nautilus sidebar is stationary, can not change it's size

**Comment by froggy-jpg** [comment]
bummer, closing this issue then, thanks anyway
